fk/teamcreate.go

```diff
@@ -176,7 +176,7 @@ func teamCreate() exitCode {
 		printCheckboxFailure("Create and sign team roster", err)
 	}
 	printCheckboxSuccess("Create and sign team roster in \n" +
-		filepath.Join(fluidkeysDirectory, "teams"))
+		"         " + filepath.Join(fluidkeysDirectory, "teams")) // account for checkbox indent
```

Comment:
I've added a tech debt card to provide a second argument to `printCheckboxSuccess` to take a second `lines` argument since we use this pattern quite a few times.